libs/Texture/Texture.cpp

```diff
@@ -51,6 +51,7 @@ int Texture::Load2DTexture()
 
 int Texture::Load3DTexture(const std::vector<std::string> &paths)
 {
+	#if (!defined(__APPLE__))
 	if (paths.size() == 0)
```

Comment:
I'd run everything through the linter again to fix the tabbing here. Is it usually `#if` or `#IF`? Whatever's best practice

Reply:
There is no rule about preprocesor directives. The convention is to CAPS the name of the macros.  Let's leave `#if`.
I ran the formatter.